frontend/src/components/ShootWorkers/ManageWorkers.vue

```diff
@@ -187,28 +237,27 @@ export default {
       this.valid = valid
       this.$emit('valid', this.valid)
     },
-    setWorkersData ({ workers, cloudProfileName, region, zonesNetworkConfiguration, updateOSMaintenance, zonedCluster, isNewCluster }) {
+    setWorkersData ({ workers, cloudProfileName, region, zonesNetworkConfiguration, updateOSMaintenance, zonedCluster, existingWorkerCIDR }) {
       this.cloudProfileName = cloudProfileName
       this.region = region
       this.zonesNetworkConfiguration = zonesNetworkConfiguration
       this.updateOSMaintenance = updateOSMaintenance
       this.setInternalWorkers(workers)
       this.zonedCluster = zonedCluster
-      this.isNewCluster = isNewCluster
+      this.isNewCluster = !existingWorkerCIDR
+      this.existingWorkerCIDR = existingWorkerCIDR
     }
   },
   mounted () {
     if (this.userInterActionBus) {
       this.userInterActionBus.on('updateCloudProfileName', cloudProfileName => {
         this.internalWorkers = []
         this.cloudProfileName = cloudProfileName
-        const cloudProfile = this.cloudProfileByName(cloudProfileName)
         /*
-         * do not pass shootspec as we have it not available in this component and it is (currently) not required
-         * do dtermine isZoned for new clusters. This event handler is only be called for new clusters, as the
+         * do not pass shootspec as we have it not available in this component and it is (currently) not required to determined isZoned for new clusters. This event handler is only be called for new clusters, as the
```

Comment:
not sure I can follow

Reply:
This is zonedCluster stuff is nothing new and required for edge cases, e.g. azure where we only support zoned clusters by the dashboard, however not zoned clusters might exists (using availability sets). We can discuss this again, but I would first need to go through the code by myself. Shall I do that, so that we can discuss this handling?

Reply:
no it's ok